### PR TITLE
Update the SVG Icons article

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

File: concepts/60 Themes and Styles/30 Icons/60 SVG Icons.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
-You can use SVG icons instead of font icons. The following code demonstrates how to use them in the [Button](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Button/Icons) widget. The same technique can be used for widgets with the [icon](/api-reference/10%20UI%20Widgets/dxButton/1%20Configuration/icon.md '/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#icon') option. 
+In addition to font icons, DevExtreme supplies the same icons in the SVG format. You can find SVG icons in the <a href="https://github.com/DevExpress/DevExtreme/tree/20_2/images/icons" target="_blank">DevExtreme</a> repository on GitHub.
```
Comment (AlexJameson) [inline concepts/60 Themes and Styles/30 Icons/60 SVG Icons.md]:
```suggestion
In addition to font icons, DevExtreme supplies the same icons in the SVG format. You can find SVG icons in the <a href="https://github.com/DevExpress/DevExtreme/tree/20_2/images/icons" target="_blank">DevExtreme repository</a> on GitHub.
```

**Review by AlexJameson — APPROVED** [review]

**Review by OlegKipchatov — APPROVED** [review]

File: concepts/60 Themes and Styles/30 Icons/10 Built-In Icon Library.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+DevExtreme comes with an icon library that contains SVG and font icons for all DevExtreme themes. You can use the icons [in widgets](/concepts/60%20Themes%20and%20Styles/30%20Icons/20%20Icons%20in%20Widgets.md '/Documentation/Guide/Themes_and_Styles/Icons/#Icons_in_Widgets') and [in other page elements](/concepts/60%20Themes%20and%20Styles/30%20Icons/30%20Icons%20in%20Other%20HTML%20Elements.md '/Documentation/Guide/Themes_and_Styles/Icons/#Icons_in_Other_HTML_Elements') as they are or [customize them](/concepts/60%20Themes%20and%20Styles/30%20Icons/35%20Customize%20Icons.md '/Documentation/Guide/Themes_and_Styles/Icons/#Customize_Icons').
 
 Available icons are presented below:
```
Comment (albertov05) [inline concepts/60 Themes and Styles/30 Icons/10 Built-In Icon Library.md]: I'd like to revise this sentence without using "presented". Can we say the following instead?

```suggestion
The following icons are available:
```

File: concepts/60 Themes and Styles/30 Icons/60 SVG Icons.md [hunk 3]
```diff
@@ -1,27 +1,27 @@
-You can use SVG icons instead of font icons. The following code demonstrates how to use them in the [Button](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Button/Icons) widget. The same technique can be used for widgets with the [icon](/api-reference/10%20UI%20Widgets/dxButton/1%20Configuration/icon.md '/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#icon') option. 
+In addition to font icons, DevExtreme supplies the same icons in the SVG format. You can find SVG icons in the <a href="https://github.com/DevExpress/DevExtreme/tree/20_2/images/icons" target="_blank">DevExtreme repository</a> on GitHub.
 
-An SVG icon can be specified as follows:
+The following code demonstrates how to use SVG icons in the [Button](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Button/Icons) widget. The same technique can be used with any other UI component that has the [icon](/api-reference/10%20UI%20Widgets/dxButton/1%20Configuration/icon.md '/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#icon') property.
```
Comment (albertov05) [inline concepts/60 Themes and Styles/30 Icons/60 SVG Icons.md]:
```suggestion
The following code uses SVG icons in the [Button](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Button/Icons) widget. The same technique can be used with any other UI component that has the [icon](/api-reference/10%20UI%20Widgets/dxButton/1%20Configuration/icon.md '/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#icon') property. 
```

File: concepts/60 Themes and Styles/30 Icons/60 SVG Icons.md [hunk 4]
```diff
@@ -1,27 +1,27 @@
...
+1. **Import the icon**:
 
         <!-- tab: JavaScript -->
         import * as myIcon from "./assets/icon.svg";
         new DevExpress.ui.dxButton(targetElement, {
             icon: myIcon
         });
 
-[important]The SVG format allows you to run executable code that might be malicious. It is strongly recommended that you use SVG icons only from trusted sources.
+[important]The SVG format allows running executable code that might be malicious. We strongly recommend that you use SVG icons only from trusted sources.
```
Comment (albertov05) [inline concepts/60 Themes and Styles/30 Icons/60 SVG Icons.md:27]: 1) `The SVG format allows running executable code that might be malicious.`

The original sentence specified _who_ ("you") could potentially run malicious code. By changing it to the -ing form, we lose this.
Can we go back to the original sentence ("allows you to run executable code")?

2) `We strongly recommend that you use SVG icons only from trusted sources.`

This sentence is fine.

File: concepts/60 Themes and Styles/30 Icons/10 Built-In Icon Library.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
...
+DevExtreme comes with an icon library that contains SVG and font icons for all DevExtreme themes. You can use the icons [in widgets](/concepts/60%20Themes%20and%20Styles/30%20Icons/20%20Icons%20in%20Widgets.md '/Documentation/Guide/Themes_and_Styles/Icons/#Icons_in_Widgets') and [in other page elements](/concepts/60%20Themes%20and%20Styles/30%20Icons/30%20Icons%20in%20Other%20HTML%20Elements.md '/Documentation/Guide/Themes_and_Styles/Icons/#Icons_in_Other_HTML_Elements') as they are or [customize them](/concepts/60%20Themes%20and%20Styles/30%20Icons/35%20Customize%20Icons.md '/Documentation/Guide/Themes_and_Styles/Icons/#Customize_Icons').
```
Comment (albertov05) [inline concepts/60 Themes and Styles/30 Icons/10 Built-In Icon Library.md]:
```suggestion
DevExtreme includes an icon library with SVG and font icons for all DevExtreme themes. You can use an icon [in widgets](/concepts/60%20Themes%20and%20Styles/30%20Icons/20%20Icons%20in%20Widgets.md '/Documentation/Guide/Themes_and_Styles/Icons/#Icons_in_Widgets') and [in other page elements](/concepts/60%20Themes%20and%20Styles/30%20Icons/30%20Icons%20in%20Other%20HTML%20Elements.md '/Documentation/Guide/Themes_and_Styles/Icons/#Icons_in_Other_HTML_Elements') as is or [customize it](/concepts/60%20Themes%20and%20Styles/30%20Icons/35%20Customize%20Icons.md '/Documentation/Guide/Themes_and_Styles/Icons/#Customize_Icons').
```

File: concepts/60 Themes and Styles/30 Icons/60 SVG Icons.md (hunk 4 above)
Comment (RomanTsukanov) [reply concepts/60 Themes and Styles/30 Icons/60 SVG Icons.md:27]: > The original sentence specified who ("you") could potentially run malicious code. By changing it to the -ing form, we lose this.

That was the motivation behind this change. It is not a common practice to put any executable code in SVG. So, if an SVG icon contains code, is is most likely malicious, and the reader never wants to run it.

Comment (albertov05) [reply concepts/60 Themes and Styles/30 Icons/60 SVG Icons.md:27]: Thanks for the clarification. I understand now. Can we say the following?

`The SVG format could allow potentially malicious code to run. We strongly recommend that you use SVG icons only from trusted sources.`

**Pull request merged**